Automattic/wp-calypso
# Stats: Updating hover indicator for Subscribers chart

Related to #75932

## Proposed Changes

* updating the look for chart data points while on hover

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Navigate to the page and verify that chart points look like on the screenshot below (when you hover your mouse over them)

| Before | After |
| --- | --- |
| <img width="1081" alt="SCR-20230510-kzmj" src="https://github.com/Automattic/wp-calypso/assets/112354940/d5273243-b803-4d91-b0ef-5ad23f4622ab"> | <img width="892" alt="SCR-20230510-kzqk" src="https://github.com/Automattic/wp-calypso/assets/112354940/db20c3bf-b7e7-48b7-87fd-9903315c6b4c"> |


## Pre-merge Checklist
